sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -472,6 +525,75 @@ void Scheduler::cleanupCommands(const std::vector<Command *> &Cmds) {
   }
 }
 
+void Scheduler::deferMemObjRelease(const std::shared_ptr<SYCLMemObjI> &MemObj) {
+  {
+    std::lock_guard<std::mutex> Lock{MDeferredMemReleaseMutex};
+    MDeferredMemObjRelease.push_back(MemObj);
+  }
+  cleanupDeferredMemObjects(false);
+}
+
+inline bool Scheduler::isNoDeferredMemObjects() {
+  std::lock_guard<std::mutex> Lock{MDeferredMemReleaseMutex};
+  return MDeferredMemObjRelease.empty();
+}
+
+void Scheduler::cleanupDeferredMemObjects(bool ForceWait) {
```

Comment:
```suggestion
void Scheduler::cleanupDeferredMemObjects(bool Blocking) {
```
? Just for consistency.

Reply:
fixed here https://github.com/intel/llvm/pull/6837/commits/b5e85de484fb026152cfcaa8d3dd8bbc79c158d3